templates/definition/charger/tasmota.yaml

```diff
@@ -7,15 +7,18 @@ params:
   - name: user
   - name: password
     mask: true
+  - name: channel
```

Comment:
Hilfstext ergänzen? Min/Max sollte ebenfalls möglich sein. Verrät das Device, wieviele Channels es hat?

Reply:
> Verrät das Device, wieviele Channels es hat?

Leider nein. Gibt keine Anzahl aktiver Relay Channels.
Aber ich leite die Anzahl der Schalt-Kanäle mal über die Existenz der POWER(X) Felder in der StatusSTS Response ab ... (siehe unten).

```json
   "StatusSTS":{
      "Time":"2022-05-13T20:54:37",
      "Uptime":"0T05:41:40",
      "UptimeSec":20500,
      "Heap":24,
      "SleepMode":"Dynamic",
      "Sleep":50,
      "LoadAvg":19,
      "MqttCount":1,
      "POWER1":"ON",
      "POWER2":"ON",
      "Wifi":{
         "AP":1,
         "SSId":"iotatbatchovski",
         "BSSId":"D2:21:F9:17:BC:41",
         "Channel":9,
         "Mode":"11n",
         "RSSI":70,
         "Signal":-65,
         "LinkCount":1,
         "Downtime":"0T00:00:04"
      }
```

Reply:
Configure gibt jetzt folgendes im Fehlerfall aus:
```bash
- Konfiguration Wallbox

 Wähle eine Wallbox: Tasmota [Schaltbare Steckdosen]

Führe folgende Einstellungen durch:

 IP-Adresse oder Hostname 192.168.178.95
 Benutzerkonto 
 Passwort [? for help] 
 Nummer des Schalt-Kanals 2
 Standby-Leistung in W 15

Teste die Konfiguration von Tasmota [Schaltbare Steckdosen] ...
   Fehler: cannot create charger 'tasmota': configured relay channel doesn't exist on device

 Der Test von Tasmota [Schaltbare Steckdosen] ist fehlgeschlagen. Soll es trotzdem in die Konfiguration aufgenommen werden? No
...
```

Reply:
Sehr cool!